securedrop/tests/test_unit_journalist.py

```diff
@@ -415,27 +418,115 @@ def test_edit_hotp(self):
         # should redirect to verification page
         self.assertRedirects(res, url_for('account_new_two_factor'))
 
-    def test_bulk_download(self):
+    def test_download_selected_from_source(self):
         sid = 'EQZGCJBRGISGOTC2NZVWG6LILJBHEV3CINNEWSCLLFTUWZJPKJFECLS2NZ4G4U3QOZCFKTTPNZMVIWDCJBBHMUDBGFHXCQ3R'
         source = Source(sid, crypto_util.display_id())
         db_session.add(source)
         db_session.commit()
-        files = ['1-abc1-msg.gpg', '2-abc2-msg.gpg']
-        filenames = common.setup_test_docs(sid, files)
+        files = ['1-abc1-msg.gpg', '2-abc2-msg.gpg', '3-abc3-msg.gpg', '4-abc4-msg.gpg']
```

Comment:
You can adapt `add_source_and_submissions()` in #1440 (once it's merged in) here